spdx-utils/src/main/kotlin/SpdxUtils.kt

```diff
@@ -70,14 +70,17 @@ fun calculatePackageVerificationCode(sha1sums: List<String>, excludes: List<Stri
  * [1]: https://spdx.org/spdx_specification_2_0_html#h.2p2csry
  */
 @JvmName("calculatePackageVerificationCodeForFiles")
-fun calculatePackageVerificationCode(files: List<File>, excludes: List<String> = emptyList()): String {
-    val digest = MessageDigest.getInstance("SHA-1")
-    return calculatePackageVerificationCode(
-        files.map { file ->
-            file.inputStream().use { digest.digest(it.readBytes()).toHexString() }
-        },
-        excludes
-    )
+fun calculatePackageVerificationCode(files: List<File>, excludes: List<String> = emptyList()): String =
+    calculatePackageVerificationCode(files.map { sha1sum(it) }, excludes)
+
+/**
+ * Return the SHA-1 sum of the given file as hex string.
+ */
+private fun sha1sum(file: File): String {
+    file.inputStream().use {
+        val digest = MessageDigest.getInstance("SHA-1")
+        return digest.digest(it.readBytes()).toHexString()
+    }
```

Comment:
Now we could make this an expression function again, similar to like we had it before e71111fad700d51db28cc061575a71984629345c:

```
private fun sha1sum(file: File) =
    MessageDigest.getInstance("SHA-1").let { digest ->
        file.inputStream().use { digest.digest(it.readBytes()).toHexString() }
    }
```

Note: The downside of this approach is that `getInstance("SHA-1")` is called for each file.

Reply:
Done. Was aware of that `downside` but I don't think it is an issue.

Reply:
Did you intentionally not use my proposed code? Because I deliberately used the way it was written before e71111fad700d51db28cc061575a71984629345c as that avoids the separate `val digest`.

Reply:
Yes, it wasn't clear to me that you wanted me to use the proposed code actually.
I thought your main point was to make it an expression function.

Reply:
Well, if I take the effort to write code ready for copy & paste, there's of course some expectation involved for it to be used :wink:

But in this case I'm fine with your solution as it avoids a nesting level in the next commit.